### PR TITLE
chore(helper-plugin)!: move RBAC / remove auth / remove usePersistentState

### PR DESCRIPTION
**Pull request by joshuaellis**
### What does it do?

* moves RBAC
* removes auth
* removes usePersistentState 

### Related issue(s)/PR(s)

* resolves CONTENT-1811
* resolves CONTENT-2101
* resolves CONTENT-2100

### CODE REVIEW

**Review by Marc-Roig — APPROVED** [review]
one day you will tell me the truth about the hidden clones of yours you have hidden in your basement

**Review by simotae14 — APPROVED** [review]
👏🏽

**Comment by jhoward1994** [comment]
> one day you will tell me the truth about the hidden clones of yours you have hidden in your basement

+1

File: packages/core/admin/vite.config.mjs
Comment (joshuaellis) [inline packages/core/admin/vite.config.mjs:1]: I've added this, i didn't want too, but i want to seal this PR, the JIRA ticket is how to remove it so i'm not too worried.

**Pull request merged**